src/pages/events.js

```diff
@@ -45,36 +48,88 @@ const dallasLogoStyles = {
 };
 
 // Component Definition
-const IndexPage = () => {
-  return (
-    <Layout>
-      <div css={rootStyles}>
-        <section css={heroContainerStyles}>
-          <header>
-            <h1 css={heroTitleStyles}>Events</h1>
-            <div css={heroTextStyles}>Where DFW meets</div>
-          </header>
-        </section>
-
-        <section css={contentStyles}>
-          <h3>December 2018</h3>
-          <h4>ReactJS @ BottleRocket</h4>
-          <div>6:15 - 7:00pm: Pizza/Drinks & meeting other people</div>
-          <div>6:30 - 7:00pm: Jobs Open Mic and jobs follow-up</div>
-          <div>7:00 - 7:45pm - Morgan Dedmon - "WASM: What is that?"</div>
-          <div>8:00 - 8:45pm: Salvador Aceves - "Redux Sagas in Practice"</div>
-
-          <p style={{ marginTop: 12 }}>
-            <a href="https://www.meetup.com/ReactJSDallas/events/pbbdwnyxqbpb/">Meetup link</a>
-          </p>
-        </section>
-
-        <div css={dallasLogoContainerStyles}>
-          <DallasLogoSvg style={dallasLogoStyles} />
-        </div>
-      </div>
-    </Layout>
-  );
-};
+export default class IndexPage extends React.Component {
+  
+  //Setup default state for our index page
+  constructor(props)
+  {
+    super(props);
+    this.state = {
+      events: [] //we will fill this after fetch
+    };
+
+
+  }
+
+  //Fetches from meetup API and stores our events
+  componentDidMount ()
+  {
+    //console.log('mounted');
+
+    //Fetch all events from our React JS meetup group!
+    fetch("https://api.meetup.com/reactjsdallas/events?key=1570558171e5f7833a5f7b1d682813", {
```

Comment:
If we need to add the key in the url here, then we should use a string template and insert the `.env` variable for it. When I run this, I get a CORS error b/c this id is hard-coded here.

Therefore, I cannot see any of this UI.